pkg/kubectl/resource/builder.go

```diff
@@ -32,6 +32,8 @@ import (
 	"k8s.io/kubernetes/pkg/util/errors"
 )
 
+var schemaFileExtensions = []string{".json", ".stdin", ".yaml", ".yml"}
```

Comment:
I would have expected this list to be shared with the bit creates content assist for the `--filenames` flag in the commands.